src/overlays/actors/ovl_En_Guest/z_en_guest.c

```diff
@@ -50,9 +51,7 @@ static InitChainEntry sInitChain[] = {
     ICHAIN_F32(targetArrowOffset, 500, ICHAIN_STOP),
 };
 
-extern FlexSkeletonHeader D_060000F0;
-extern AnimationHeader D_060042AC;
-extern Gfx D_060059B0[];
+extern FlexSkeletonHeader object_boj_Skel_0000F0;
```

Comment:
Why is this still `extern`?

Reply:
ZAPD can't extract that skeleton because its limbs use unaligned textures

There's a comment about this in undefined_syms.txt where that symbol and others are defined